### PR TITLE
mega commit, needs some cleanups

### PR DESCRIPTION
**Pull request by kurtseifried**
please let me have 24 hours to clean this up before you actually consume it, I just wanted to get it moving

### CODE REVIEW

**Comment by lcashdol** [comment]
I can clean up my stuff if you like Kurt.

**Comment by kurtseifried** [comment]
Larry your stuff is in https://github.com/CVEProject/cvelist/pull/1316 and I need to fix a few things (my bad).

**Comment by cve-team** [comment]
Have you finished cleaning this up?

**Comment by cve-team** [comment]
Ping?

**Comment by lcashdol** [comment]
I think I've gotten all of my CVE JSON fixed up.

**Comment by cve-team** [comment]
Per a private email, work is still needed on this.

File: 2018/1000xxx/CVE-2018-1000843.json [hunk 1]
```diff
@@ -0,0 +1 @@
...sion 2.8.0; after commit 53b52e12745075a8acc016d33945d9d6a7a6aaeb; after PR https://github.com/spotify/luigi/pull/1870 contains a Cross ite Request Forgery (CSRF) vulnerability in API endpoint: /api/<method> that can result in Task metadata such as task name, id, parameter, etc. will be leaked to unauthorized users. This attack appear to be exploitable via The victim must visit a specially crafted webpage from the network where their Luigi server is accessible.. This vulnerability appears to have been fixed in 2.8.0 and later."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "prior to version 2.8.0; after commit 53b52e12745075a8acc016d33945d9d6a7a6aaeb; after PR https://github.com/spotify/luigi/pull/1870"}]},"product_name": "Luigi"}]},"vendor_name": "Luigi"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-11-27T13:54:33.480387","DATE_REQUESTED": "2018-11-02T13:25:28","ID": "CVE-2018-1000843","ASSIGNER": "kurt@seifried.org","REQUESTER": "honnix@spotify.com"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "Cross ite Request Forgery (CSRF)"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000843.json]: CVE does not support URLs such as this embedded in descriptions.

File: 2018/1000xxx/CVE-2018-1000842.json [hunk 2]
```diff
@@ -0,0 +1 @@
...dcfa"}]},"description": {"description_data": [{"lang": "eng","value": "FatFreeCRM version <=0.14.1, >=0.15.0 <=0.15.1, >=0.16.0 <=0.16.3, >=0.17.0 <=0.17.2, ==0.18.0 contains a Cross Site Scripting (XSS) vulnerability in https://github.com/fatfreecrm/fat_free_crm/commit/6d60bc8ed010c4eda05d6645c64849f415f68d65 that can result in Javascript execution. This attack appear to be exploitable via Content with Javascript payload will be executed on end user browsers when they visit the page. This vulnerability appears to have been fixed in 0.18.1, 0.17.3, 0.16.4, 0.15.2, 0.14.2."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "<=0.14.1, >=0.15.0 <=0.15.1, >=0.16.0 <=0.16.3, >=0.17.0 <=0.17.2, ==0.18.0"}]},"product_name": "FatFreeCRM"}]},"vendor_name": "FatFreeCRM"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-11-27T13:54:33.479249","DATE_REQUESTED": "2018-10-27T06:04:25","ID": "CVE-2018-1000842","ASSIGNER": "kurt@seifried.org","REQUESTER": "security@fatfreecrm.com"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "Cross Site Scripting (XSS)"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000842.json]: CVE does not support URLs such as this embedded in descriptions.

**Comment by kurtseifried** [comment]
cleaned up and ready to go I think.

**Pull request merged**